inference-engine/src/vpu/myriad_plugin/myriad_config.cpp

```diff
@@ -59,7 +59,6 @@ const std::unordered_set<std::string>& MyriadConfig::getDeprecatedOptions() cons
 IE_SUPPRESS_DEPRECATED_START
     static const std::unordered_set<std::string> options = merge(ParsedConfig::getDeprecatedOptions(), {
         VPU_MYRIAD_CONFIG_KEY(FORCE_RESET),
-        VPU_MYRIAD_CONFIG_KEY(PLATFORM),
```

Comment:
It is still deprecated

Reply:
Removed PLATFORM from getDeprecatedOptions() on purpose. As I see, this function is only used for showing the warning of deprecated option usage.
I did it so that users don't get the warning every time, because they cannot know I used it :) 
Is it alright?

Reply:
Not yet, Is `PLATFORM` public option? If so, user may use it. Is `PLATFORM` deprecated? If so, user should see warning if `PLATFORM` is used.

Reply:
Not yet, Is PLATFORM public option? If so, user may use it. Is PLATFORM deprecated? If so, user should see warning if PLATFORM is used.

Reply:
Yes, it is the public option and it is the deprecated option. I think it must be in the list of deprecation anyway

Reply:
Okay, I can revert this change.
But this option is going to be used anyway (there is no condition for its usage), even if there is a device. Is it alright for user to get warning in this case?

Reply:
I think so

Reply:
It's not that good for user to get a warning here, but proper solution seems to be risky to implement now just before CF